FinSearchUnified/Tests/Bundle/SearchBundleFindologic/SortingHandler/PriceSortingHandlerTest.php

```diff
@@ -14,24 +14,24 @@
 class PriceSortingHandlerTest extends TestCase
 {
     /**
-     * @var QueryBuilder
+     * @var NewQueryBuilder
      */
     private $querybuilder;
 
     /**
      * @var ProductContextInterface
      */
-    private $context = null;
+    private $context;
 
     /**
      * @throws Exception
      */
     protected function setUp()
     {
         parent::setUp();
+        $_SERVER['REMOTE_ADDR'] = '192.168.0.1';
```

Comment:
Instead of always setting this, you may also just have a default in the query builder in case this is not set.

Reply:
But that is a required field. Is that OK to have a default value within the querybuilder?

Reply:
Yes in a real environment this is always set anyway.